README.md

```diff
@@ -1,2 +1,33 @@
-# loopback-sdk-ios-codegen
-Code generation tool for LoopBack iOS SDK
+# LoopBack iOS SDK CodeGen
+
+The iOS CodeGen command line tool generates iOS client side LoopBack Model representations
+in Objective-C by looking into the specified server application.
+
+## How to Use
+
+The following is an example of the usage applied to the test server provided under this directory:
+
+ 1. Run `npm install` in this directory to initialize the tool `bin/lb-ios` and the test server under `test-env/server`.
+ *  Run `bin/lb-ios -p XX test-env/server/server test-env/client/ios/gen-src` 
+ 	to generate the models in Objective-C representation, 
+ 	where `XX` is the prefix attached to all the generated class names.
+
+## How to Run the Unit Test
+
+After performing the above steps, run the followings:
+
+ 1. Run the test server by `node test-env/server/server.js &`.
+ *  Copy `LoopBack.framework` into `test-env/CodeGenTest/`.
```

Comment:
Could I deal with this a bit later once the entire structure becomes stabler?

I guess my explanation on this codegen was a bit too short and unkind.
`LoopBack.framework` is the one generated from `loopback-sdk-ios` (it is a package of the lib file and the headers).
The usage of this codegen I intended was that, instead of dealing with all the SDK in the shape of source files like Xamarin SDK, developers incorporates precompiled core part (including SLRemoting, SLAFNetworking and basic LBModel classes) as an iOS framework and compile their app with this framework together with the generated code that is specific to their app.  I thought this is simpler for the developers. 
Does this make sense?
If this is the way to go, maybe I should use CocoaPods for the dependency management.